README.rst

```diff
@@ -64,7 +64,7 @@ an abundance of metrics and information from all of the tracked leagues. The
 examples below are only a miniscule subset of the total number of statistics
 that can be pulled using sportsreference. Visit the documentation on
 `Read The Docs <http://sportsreference.readthedocs.io/en/latest/>`_ for a
-complete list of all information exposed by the API.
+complete list of all information exposed by the API.You can also check the ML directory for a machine learning example which uses this module!
```

Comment:
Missing a space between "API." and "You". Also, probably want to wrap this line to 80 characters just to be uniform with the other lines in the README.

Reply:
One more item - though I know you have the PR for the examples open, it technically isn't part of this PR, so if this PR were to be merged before the other, this line wouldn't be valid. I would keep mention of the ML directory out of this PR since it isn't related, unless the code is already in master prior to this update.